build.sh

```diff
@@ -0,0 +1,548 @@
+#!/bin/bash
+set -e
+
+#-----------------------------------------------------------------------------
+# RediSearch Build Script
+# 
+# This script handles building the RediSearch module and running tests.
+# It supports various build configurations and test types.
+#-----------------------------------------------------------------------------
+
+# Get the absolute path to script directory
+ROOT="$(cd "$(dirname "$0")" && pwd)"
+BINROOT="$ROOT/bin"
+
+#-----------------------------------------------------------------------------
+# Default configuration values
+#-----------------------------------------------------------------------------
+COORD="oss"      # Coordinator type: oss or rlec
+DEBUG=0          # Debug build flag
+FORCE=0         # Force clean build flag
+VERBOSE=0        # Verbose output flag
+QUICK=0          # Quick test mode (subset of tests)
+
+# Test configuration (0=disabled, 1=enabled)
+BUILD_TESTS=0    # Build test binaries
+RUN_UNIT_TESTS=0 # Run C/C++ unit tests
+RUN_RUST_TESTS=0 # Run Rust tests
+RUN_PYTEST=0     # Run Python tests
+RUN_ALL_TESTS=0  # Run all test types
```

Comment:
i would like to have the run configurations as flags and not as boolean inputs, is it possible?